source/SIL.AppBuilder.Portal.Frontend/src/translations/index.tsx

```diff
@@ -1,10 +1,8 @@
 import i18n from 'i18next';
-import * as ICU from 'i18next-icu';
-import { reactI18nextModule } from 'react-i18next';
-import * as LanguageDetector from 'i18next-browser-languagedetector';
-
-// import * as Backend from 'i18next-xhr-backend';
-// import { reactI18nextModule } from 'react-i18next';
+import ICU from 'i18next-icu';
```

Comment:
result of some upgrading as I tried to use i18next to do things for me.
supposedly it supports async/remote translations, but the lack of fetch/xhr invocation told me otherwise.

that's why `fetch-l10n.tsx` exists now. :(